spec/cirro_io_v2/resources/gig_spec.rb

```diff
@@ -14,8 +14,8 @@
     end
 
     it 'returns gig structure' do
-      expect(create_gig).to be_a(Struct)
-      expect(create_gig.to_h).to eq(gig.to_h)
+      expect(create_gig).to be_a(CirroIOV2::Responses::GigResponse)
```

Comment:
Is there a test where you check the behavior when you receive a list of gig-invitations?

https://developers.cirro.io/api-docs/v1#get-a-list-of-gig-invitations

I would suggest to check this use case as well.